scripts/constant/addresses.ts

```diff
@@ -0,0 +1,41 @@
+export default {
+  mainnet: {
+    connectors: {
+      basic: "0xe5398f279175962E56fE4c5E0b62dc7208EF36c6",
+      auth: "0xd1aff9f2acf800c876c409100d6f39aea93fc3d9",
```

Comment:
@pradyuman-verma Can you check if we are using it anywhere?

Reply:
If not, we can remove or replace like this:
```suggestion
      "BAISC-A": "0x9926955e0dd681dc303370c52f4ad0a4dd061687",
      "AUTHORITY-A": "0x351bb32e90c35647df7a584f3c1a3a0c38f31c68",
```

Reply:
Same for other networks

Reply:
no, we are not using them anywhere